x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -182,18 +182,43 @@ export const defineExplainLogRateSpikesRoute = (
             })
           );
 
-          let fieldCandidates: Awaited<ReturnType<typeof fetchFieldCandidates>>;
+          // Step 1: Index Info: Field candidates, total doc count, sample probability
+
+          const fieldCandidates: Awaited<ReturnType<typeof fetchIndexInfo>>['fieldCandidates'] = [];
+          let sampleProbability = 1;
+          let totalDocCount = 0;
+
+          logDebugMessage('Fetch field candidates.');
+          push(
+            updateLoadingStateAction({
+              ccsWarning: false,
+              loaded,
+              loadingState: i18n.translate(
+                'xpack.aiops.explainLogRateSpikes.loadingState.loadingFieldCandidates',
+                {
+                  defaultMessage: 'Loading field candidates.',
+                }
```

Comment:
Are we loading field candidates twice? Seems like a repeat of line 179.

Reply:
Thanks for spotting this, looks like I made this mistake when cherry-picking from another draft PR. Fixed in [4ffeab3](https://github.com/elastic/kibana/pull/143977/commits/4ffeab374871e81fafa63bb52df0fd986656b23b).